api/src/controllers/courtCases.ts

```diff
@@ -0,0 +1,20 @@
+import type { Response } from "express"
+import type { DataSource } from "typeorm"
+import getDataSource from "../services/getDataSource"
+import listCourtCases from "../services/listCourtCases"
+import type { CaseListQueryRequest } from "../../src/types/CaseListQueryRequest"
+
+export const getCourtCases = async (req: CaseListQueryRequest, res: Response) => {
+  const { caseListQueryParams } = req
+  const dataSource: DataSource = await getDataSource()
+  try {
+    const data = await listCourtCases(dataSource, caseListQueryParams!)
```

Comment:
What would happen if caseListQueryParams is undefined?

Reply:
listCourtCases would throw an error. The validator middleware would bounce it back before this stage though, so I thought it was safe enough.